### DISCUSSION TITLE
iPhone route are under router IP. Not iPhone IP.

### DISCUSSION THREAD

**Discussion by kuduacz** [AdguardTeam/AdGuardHome#4864] [Q&A]
I have a AHG as homeassistant addon. 
In router set DNS to AGH and in phones too. When i visit GitHub.com for example in AGH is in under router IP not iPhone. 
But when i visit same site on wife's android phone with same settings its under her IP in AGH. 
Why and how to switch i on my iPhone?
iCloud private relay is off.

**Comment by kuduacz** [comment]
A other thing. 
When i restart network settings its ok. Requests od under iPhone ip, bot router. 
When i leave home, and back (do when i disconnect to wifi and connect back) all request grom iPhone od under router IP again in adguard home.

**Reply by joshqou** [reply]
Do any other devices behave like this?

**Reply by kuduacz** [reply]
no but im preety shure that i found culprit od that behave. Tailscale (VPN what im using to acces HA when im not in home) work even when im in home so all requests go throu tailscale to router and then to adh. 
When i disabled it everything work fine.
